Index: Makefile
===================================================================
--- Makefile	(revision 380313)
+++ Makefile	(working copy)
@@ -3,7 +3,7 @@
 
 PORTNAME=		radicale
 PORTVERSION=		0.8
-PORTREVISION=		1
+PORTREVISION=		2
 CATEGORIES=		www python
 MASTER_SITES=		CHEESESHOP
 PKGNAMEPREFIX=		${PYTHON_PKGNAMEPREFIX}
@@ -34,7 +34,7 @@
 .endif
 
 .if ${PORT_OPTIONS:MHTTP}
-RUN_DEPENDS+=	${PYTHON_PKGNAMEPREFIX}requests1>=0:${PORTSDIR}/www/py-requests1	
+RUN_DEPENDS+=	${PYTHON_PKGNAMEPREFIX}requests1>=0:${PORTSDIR}/www/py-requests1
 .endif
 
 .if ${PORT_OPTIONS:MLDAP}
@@ -47,6 +47,7 @@
 
 post-install:
 	${MKDIR} ${STAGEDIR}/${ETCDIR}
+	${MKDIR} ${STAGEDIR}/${DATADIR}
 	${INSTALL_DATA} ${WRKSRC}/config ${STAGEDIR}/${ETCDIR}/config.sample
 	${INSTALL_DATA} ${WRKSRC}/logging ${STAGEDIR}/${ETCDIR}/logging.sample
 
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 380313)
+++ pkg-plist	(working copy)
@@ -1,6 +1,6 @@
 bin/radicale
-%%ETCDIR%%/config.sample
-%%ETCDIR%%/logging.sample
+@sample %%ETCDIR%%/config.sample
+@sample %%ETCDIR%%/logging.sample
 %%PYTHON_LIBDIR%%/site-packages/radicale/__init__.py
 %%PYTHON_LIBDIR%%/site-packages/radicale/__init__.pyc
 %%PYTHON_LIBDIR%%/site-packages/radicale/__init__.pyo
@@ -61,11 +61,4 @@
 %%PYTHON_LIBDIR%%/site-packages/radicale/xmlutils.py
 %%PYTHON_LIBDIR%%/site-packages/radicale/xmlutils.pyc
 %%PYTHON_LIBDIR%%/site-packages/radicale/xmlutils.pyo
-@dirrmtry %%ETCDIR%%
-@dirrm %%PYTHON_LIBDIR%%/site-packages/radicale
-@dirrm %%PYTHON_LIBDIR%%/site-packages/radicale/auth
-@dirrm %%PYTHON_LIBDIR%%/site-packages/radicale/rights
-@dirrm %%PYTHON_LIBDIR%%/site-packages/radicale/storage
-@dirrmtry %%PYTHON_LIBDIR%%/site-packages
-@dirrmtry %%PYTHON_LIBDIR%%
-@exec /usr/bin/install -d -o radicale %%DATADIR%%
+@dir(radicale,radicale,0755) %%DATADIR%%
