Index: Makefile
===================================================================
--- Makefile	(revision 382444)
+++ Makefile	(working copy)
@@ -2,7 +2,7 @@
 # $FreeBSD$
 
 PORTNAME=	php55
-PORTVERSION=	5.5.22
+PORTVERSION=	5.5.23
 PORTREVISION?=	0
 CATEGORIES?=	lang devel www
 MASTER_SITES=	${MASTER_SITE_PHP}
Index: distinfo
===================================================================
--- distinfo	(revision 382444)
+++ distinfo	(working copy)
@@ -1,4 +1,4 @@
-SHA256 (php-5.5.22.tar.bz2) = c218c184bef2905bc79fcdda6040f3d1738261395fb706396935d1c6f6e162bb
-SIZE (php-5.5.22.tar.bz2) = 13305486
+SHA256 (php-5.5.23.tar.bz2) = a99ab264dcd40181baa9defeaa4b21eb2c20d4e9316b904cc05f628762e6ada7
+SIZE (php-5.5.23.tar.bz2) = 13308115
 SHA256 (php-5.5.x-mail-header.patch) = b0b5a7c961b2052eb14d9528e76155cbeaa881fb9b4a49f452f9dab07b6fb1c4
 SIZE (php-5.5.x-mail-header.patch) = 3379
