Index: Makefile
===================================================================
--- Makefile	(revision 385593)
+++ Makefile	(working copy)
@@ -2,6 +2,7 @@
 
 PORTNAME=	zope213
 PORTVERSION=	2.13.22
+PORTREVISION=	1
 CATEGORIES=	www python zope
 MASTER_SITES=	CHEESESHOP
 DISTNAME=	Zope2-${PORTVERSION}
@@ -81,9 +82,9 @@
 
 USES=		python:2 zip
 USE_PYTHON=	distutils autoplist
+NO_ARCH=	yes
 
-PKGMESSAGE=	${WRKDIR}/pkg-message
-SUB_FILES=	instance-message package-pkg-message pkg-message
+SUB_FILES=	pkg-message
 SUB_LIST=	ZOPEINSTANCEBASE=${ZOPEINSTANCEBASE} \
 		ZOPE_USER=${ZOPE_USER} \
 		ZOPEINSTANCEDIR=${ZOPEINSTANCEDIR}
@@ -93,30 +94,7 @@
 ZOPEINSTANCEBASE?=	${PREFIX}/www/Zope213
 ZOPEINSTANCEDIR?=	${ZOPEINSTANCEBASE}/${ZOPEINSTANCENAME}
 
-.if defined(BATCH)
-ISBATCH=	--user admin:test123
-.else
-ISBATCH=	""
-.endif
+post-patch:
+	@${REINPLACE_CMD} -e 's/#.*effective-user chrism/effective-user ${ZOPE_USER}/' ${WRKSRC}/src/Zope2/utilities/skel/etc/zope.conf.in
 
-.if defined(INSTANCENAME)
-ZOPEINSTANCENAME=	${INSTANCENAME}
-.endif
-
-instance:
-	@${LOCALBASE}/bin/mkzopeinstance --dir ${ZOPEINSTANCEDIR} ${ISBATCH}
-	@${CHOWN} ${ZOPE_USER}:${ZOPE_USER} ${ZOPEINSTANCEDIR}/var
-	@${CHOWN} ${ZOPE_USER}:${ZOPE_USER} ${ZOPEINSTANCEDIR}/log
-	@${SED} ${SUB_LIST:S/$/!g/:S/^/ -e s!%%/:S/=/%%!/} \
-		${FILESDIR}/zope.conf_changes \
-		| ${PATCH} --silent ${ZOPEINSTANCEDIR}/etc/zope.conf
-	@${RM} ${ZOPEINSTANCEDIR}/etc/zope.conf.orig
-	@${CAT} ${WRKDIR}/instance-message
-
-post-install:
-	@${CAT} ${WRKDIR}/pkg-message
-
-pre-package:
-	@${INSTALL_DATA} ${WRKDIR}/package-pkg-message ${PKGMESSAGE}
-
 .include <bsd.port.mk>
Index: files/instance-message.in
===================================================================
--- files/instance-message.in	(revision 385593)
+++ files/instance-message.in	(working copy)
@@ -1,16 +0,0 @@
-**********************************************************************
-
-Zope instance successfully installed
-
-If Zope should get started automatically when the system starts,
-please add the following line to /etc/rc.conf
-
-zope213_enable="YES"
-
-If you are using not default instance location, please add this
-line to /etc/rc.conf to make it working:
-
-zope213_instances="%%ZOPEINSTANCEDIR%%"
-
-If there is already a zope213_instances entry please add
-%%ZOPEINSTANCEDIR%% separated by a space.
Index: files/package-pkg-message.in
===================================================================
--- files/package-pkg-message.in	(revision 385593)
+++ files/package-pkg-message.in	(working copy)
@@ -1,28 +0,0 @@
-**********************************************************************
-
-Zope Successfull installed.
-
-To create an instance of Zope please run
-%%LOCALBASE%%/bin/mkzopeinstance --dir <INSTANCEDIR>
-
-<INSTANCEDIR> is the directory where the instance files should
-              be installed. This can be %%ZOPEINSTANCEBASE%%
-
-Command-line options to 'mkzopeinstance' are available, and can be
-investigated by running 'mkzopeinstance --help'.
-
-Zope instances, created by mkzopeinstance, will not be removed
-when the package is deinstalled. You'll have to delete them yourself.
-
-If Zope should get started automatically when the system starts,
-please add the following line to /etc/rc.conf
-
-zope213_enable="YES"
-
-If you are using not default instance location, please add this
-line to /etc/rc.conf to make it working:
-
-zope213_instances="<INSTANCEDIR>"
-
-Additional Zope instances have to be added to zope213_instances
-separated by a space.
Index: files/pkg-message.in
===================================================================
--- files/pkg-message.in	(revision 385593)
+++ files/pkg-message.in	(working copy)
@@ -2,22 +2,27 @@
 
 Zope Successfull installed.
 
-To create an instance of Zope please run
-make instance
+To create an instance of Zope using the default location, please run:
 
-You can set the following Variables in /etc/make.conf
+%%LOCALBASE%%/bin/mkzopeinstance --dir %%ZOPEINSTANCEDIR%%
+chown %%ZOPE_USER%%:%%ZOPE_USER%% %%ZOPEINSTANCEDIR%%var
+chown %%ZOPE_USER%%:%%ZOPE_USER%% %%ZOPEINSTANCEDIR%%log
 
-ZOPEINSTANCEBASE	Directory where Zope instance should go
-			(default: %%ZOPEINSTANCEBASE%%)
-ZOPEINSTANCENAME	Default name for a new Zope instance
-			(default: none)
+Command-line options to 'mkzopeinstance' are available, and can be
+investigated by running 'mkzopeinstance --help'.
 
-If you want have multiple Zope you should run
-make instance INSTANCENAME=<somename>
+Zope instances, created by mkzopeinstance, will not be removed
+when the package is deinstalled. You'll have to delete them yourself.
 
-You may also define BATCH option if you want instance to be created with
-default username/password pair for instance administartor (admin:test123):
-make instance BATCH=yes
+If you are not using the default instance location, please use the
+desired paths during creation and add this line to /etc/rc.conf:
 
-Zope instances, created by `make instance', will not be removed when the
-port is deinstalled. You'll have to delete them yourself.
+zope213_instances="<INSTANCEDIR>"
+
+Additional Zope instances have to be added to zope213_instances
+separated by a space.
+
+If Zope should get started automatically when the system starts,
+please add the following line to /etc/rc.conf:
+
+zope213_enable="YES"
Index: files/zope.conf_changes
===================================================================
--- files/zope.conf_changes	(revision 385593)
+++ files/zope.conf_changes	(working copy)
@@ -1,10 +0,0 @@
---- zope.conf.in	2011-11-30 15:30:18.000000000 +0400
-+++ zope.conf	2011-11-30 15:43:40.000000000 +0400
-@@ -161,6 +161,7 @@
- #
- #    effective-user chrism
- 
-+effective-user %%ZOPE_USER%%
- 
- # Directive: enable-product-installation
- #
